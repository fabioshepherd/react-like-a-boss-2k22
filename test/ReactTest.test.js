import renderer from 'react-test-renderer';
it('Hover Test', () => {
    const component = renderer.create(
     <span id='firstSpan' onClick={() => {console.log("test2")}}> TEST </span>
      );

      let json = component.toJSON();
      expect(json).toMatchSnapshot();

      renderer.act(() => {
        json.props.onClick();
      }); 

      json = component.toJSON();
      expect(json).toMatchSnapshot();
});