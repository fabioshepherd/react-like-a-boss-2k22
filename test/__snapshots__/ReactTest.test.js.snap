// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle Test 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-cc5o4n{padding:16px 0;margin-bottom:24px;text-transform:capitalize;}",
      }
    }
    data-emotion="css cc5o4n"
  />,
  <div
    className="css-cc5o4n"
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-gepadz-MuiTypography-root{margin:0;font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:400;font-size:3rem;line-height:1.167;letter-spacing:0em;}",
        }
      }
      data-emotion="css gepadz-MuiTypography-root"
    />
    <h3
      className="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
    >
      Prova
    </h3>
  </div>,
]
`;
